teamtemp/templates/about.html

```diff
@@ -1,27 +1,30 @@
 {% extends "base.html" %}
 
-{% block title %}About Team Temperature{% endblock %}
+{% block title %}About Team Temp{% endblock %}
 
 {% block body %}
-
+<div class="container">
```

Comment:
Should this be in the base template?

All body content will be in a container, will be end up with lots of duplicate code by adding a container to each template?